explanation/intro-to/multipath.md

```diff
@@ -1,58 +1,57 @@
 
 ## Multipath setup overview
 
-Setting up multipath is often a simple procedure, since it includes compiled-in default settings that are suitable for common multipath configurations. The basic procedure for configuring your system with multipath is as follows:
+The multipath setup process is usually simple because it has compiled-in default settings that are suitable for common multipath configurations. The basic procedure for configuring your system with multipath is:
 
-1.  Install the `multipath-tools` and `multipath-tools-boot` packages
+1.  Install the `multipath-tools` and `multipath-tools-boot` packages.
 
-1.  Create an empty config file called `/etc/multipath.conf`
+2.  Create an empty config file called `/etc/multipath.conf`.
 
-1.  Edit the `multipath.conf` file to modify default values and save the updated file
+3.  Edit the `multipath.conf` file to modify default values and save the updated file.
 
-1.  Start the multipath daemon
+4.  Start the multipath daemon.
 
-1.  Update initial RAM disk
+5.  Update initial RAM disk.
 
 For detailed setup instructions for multipath configuration see {ref}`DM-Multipath configuration <configuring-multipath>` and {ref}`DM-Multipath setup <multipath-configuration-examples>`.
 
-## Multipath devices
-
-Without multipath, each path from a server node to a storage controller is treated by the system as a separate device, even when the I/O path connects the same server node to the same storage controller. Multipath provides a way of organising the I/O paths logically, by creating a single device on top of the underlying paths.
-
-### Multipath device identifiers
-
-Each multipath device has a World Wide Identifier (WWID), which is guaranteed to be globally unique and unchanging. By default, the name of a multipath device is set to its WWID. Alternatively, you can set the `user_friendly_names` option in `multipath.conf`, which causes multipath to use a **node-unique alias** of the form `mpathn` as the name.
+## Example of a multipath device in use
 
-For example, a node with two host bus adapters (HBAs) attached to a storage controller, with two ports, via a single un-zoned {term}`FC` switch sees four devices:  `/dev/sda`, `/dev/sdb`, `/dev/sdc`, and `/dev/sdd`. Multipath creates a single device with a unique WWID that reroutes I/O to those four underlying devices according to the multipath configuration.
+For example, a node with two host bus adapters (HBAs) attached to a storage controller, with two ports, via a single un-zoned {term}`FC` switch sees four devices:  `/dev/sda`, `/dev/sdb`, `/dev/sdc`, and `/dev/sdd`. If you use multipath with this node, a single multipath device is created with a unique WWID that reroutes I/O to those four underlying devices according to the multipath configuration.
```

Comment:
I prefer the original text of the second sentence, since it's written in a more active voice ("Multipath creates a single device"). This change uses the passive voice ("a single multipath device is created"), which we try to avoid wherever possible. The active voice is usually more straightforward to parse for readers who have English as a second (or +) language.